cmd/helm/serve.go

```diff
@@ -56,6 +58,7 @@ func newServeCmd(out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.StringVar(&srv.repoPath, "repo-path", helmpath.Home(homePath()).LocalRepository(), "local directory path from which to serve charts")
 	f.StringVar(&srv.address, "address", "127.0.0.1:8879", "address to listen on")
+	f.StringVar(&srv.url, "url", "", "url of chart repository")
```

Comment:
I think I'd agree that this should be `external URL of chart repository`. Since you do say URL, I wouldn't _require_ adding the `http://` part, but it wouldn't hurt.